db/sql/0_msar.sql

```diff
@@ -662,6 +662,46 @@ SELECT EXISTS (SELECT 1 FROM pg_attribute WHERE attrelid=tab_id AND attname=col_
 $$ LANGUAGE SQL RETURNS NULL ON NULL INPUT;
 
 
+CREATE OR REPLACE FUNCTION msar.get_schemas() RETURNS jsonb AS $$/*
+Return a json array of objects describing the user-defined schemas in the database.
+
+PostgreSQL system schemas are ignored.
+
+Internal Mathesar-specifc schemas are INCLUDED. These should be filtered out by the caller. This
+behavior is to avoid tight coupling between this function and other SQL files that might need to
+define additional Mathesar-specific schemas as our codebase grows.
+
+Each returned JSON object in the array will have the form:
+  {
+    "oid": <int>
+    "name": <str>
+    "description": <str|null>
+    "table_count": <int>
+  }
+*/
+SELECT jsonb_agg(schema_data)
+FROM (
+  SELECT 
+    s.oid AS oid,
+    s.nspname AS name,
+    obj_description(s.oid) AS description,
```

Comment:
I'm sorry to go around again for such a small thing, but I think you were correct in a previous conversation (or comment; can't remember) when you noted that we should namespace these things since a (perhaps foolish) user could have created a function with the same name in some context. So, it'd be better to use `pg_catalog.obj_description`.

Reply:
Cool. Makes sense. Fixed in f3289641fcf8f33ebe2c64aaf7d6c98dbf9e7bd4